build.gradle

```diff
@@ -65,7 +65,13 @@ jacocoTestCoverageVerification {
   }
 }
 
-
+configurations {
+  integrationTestCompile.extendsFrom testCompile
+  integrationTestRuntime.extendsFrom testRuntime
```

Comment:
We may not needed these 2 integration tasks as Integration tasks has serenity report configured now. These 2 line will conflict with serenity integration.